GitCommands/Settings/AppSettings.cs

```diff
@@ -1400,6 +1400,28 @@ public static bool RememberNumberOfContextLines
             set => SetBool("RememberNumberOfContextLines", value);
         }
 
+        public static bool ShowSyntaxHighlightingInDiff
+        {
+            get
+            {
+                const bool defaultValue = true;
```

Comment:
I don't believe we are achieving any gains here, but inhibiting the readability.
Please remove and convert to an expression